.github/workflows/pipy.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt-get install --yes python3-wheel twine
+
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+        run: |
+          python3 setup.py sdist bdist_wheel
+          twine upload dist/*
+      
+      - name: Publish tar.gz
+        uses: actions/upload-artifact@v3
+        path: dist/
+
+  verify:
+    runs-on: ubuntu-22.04
+    needs: publish
+    steps:
+      - name: Verify installation
+        run: |
+          python3 -m pip install prometheus-salt-exporter
```

Comment:
Now it's inconsistent, as before you where using `python`? I'm fine with both. If it works, maybe `python` is even better, but I think it should be consistent.